portal/2022-10-05-changelong.md

```diff
@@ -0,0 +1,10 @@
+---
+authors:
```

Comment:
I think this should be `authors: posdealer` (like e.g. here: https://github.com/fiskaltrust/release-notes/edit/master/portal/2020-07-20-sprint-78.md)